src/commands/run.ts

```diff
@@ -1,71 +0,0 @@
-import { isDirectoryPresent } from '@jupiterone/integration-sdk-runtime';
-import { exec } from 'child_process';
-import { createCommand } from 'commander';
-import { promises as fs } from 'fs';
-import * as path from 'path';
-import { parseConfigYaml } from '../util/parseConfig';
-import * as util from 'util';
-
-const execPromise = util.promisify(exec);
-
-export function run() {
-  return createCommand('run')
-    .description('collect and upload entities and relationships')
-    .action(async (options) => {
-      const config = await parseConfigYaml('config.yaml');
-
-      if (config.storage) {
-        //set up .env for storage configuration
-        await fs.writeFile(
-          '.env',
-          `
-          NEO4J_URI=${config.storage.config.uri}
-          NEO4J_USER=${config.storage.config.username}
-          NEO4J_PASSWORD=${config.storage.config.password}
-        `,
-        );
-      }
-
-      for (const integration of config.integrations) {
-        if (await isDirectoryPresent(integration.directory)) {
-          // Set up .env file prior to calling `yarn start`
-          let configArray = Object.keys(integration.config).map(function (
-            configEntryName,
-          ) {
-            let configArr = `${configEntryName}=${integration.config[configEntryName]}\n`;
-            return configArr;
-          });
-          await fs.writeFile(
-            path.join(integration.directory, '.env'),
-            configArray,
```

Comment:
Node.js was throwing an array writing an array to a file. 

```
TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of Array
```

I fixed the logic and extracted it into a separate function: https://github.com/JupiterOne/starbase/pull/14/files#diff-adecf0961b8a5273b6c8d737b3b9ec208dc1bc0958df2e1c6542a7bd73a2b3bbR23